src/main/resources/reference.conf

```diff
@@ -869,6 +869,29 @@ resourceTypes = {
     }
     reuseIds = false
   }
+  catalog = {
+    actionPatterns = {
+      "create_metadata" = {
+        description = "Can create catalog metadata"
+      },
+      "read_metadata" = {
```

Comment:
These action names aren't exactly accurate in that all users have permission to create/read/update metadata for any dataset based on their permission as reported by the dataset's storage system. These actions are like overrides that ignore the storage system permission setting. It's almost like `read_all_metadata` or `read_any_metadata`.

Note that these actions won't be set on an individual resource but are set on the application (`catalog`) resource.

Reply:
That's a good point on the naming!